packages/connection/src/Manager.php

```diff
@@ -139,7 +140,6 @@ public function setup_xmlrpc_handlers(
 			}
 		}
 
-		add_action( 'jetpack_clean_nonces', array( $this, 'clean_nonces' ) );
 		if ( ! wp_next_scheduled( 'jetpack_clean_nonces' ) ) {
```

Comment:
Should we move the `wp_next_scheduled()`/`wp_schedule_event()` calls up too?

Reply:
Yeah, probably. 2f36a37c9

Reply:
Actually, can we move the `add_action()` and the scheduling stuff up to `::init()`? It has nothing to do with XML-RPC.

Reply:
6318302f1